include/mbedtls/ssl.h

```diff
@@ -4496,6 +4490,16 @@ void mbedtls_ssl_conf_preference_order(mbedtls_ssl_config *conf, int order);
  */
 void mbedtls_ssl_conf_session_tickets(mbedtls_ssl_config *conf, int use_tickets);
 
+/**
+ * \brief          Get if TLS 1.2 session tickets usage is enabled or not
+ *
+ * \param conf     SSL configuration
+ *
+ * \return         MBEDTLS_SSL_SESSION_TICKETS_ENABLED or
+ *                 MBEDTLS_SSL_SESSION_TICKETS_DISABLED
+ */
+int mbedtls_ssl_conf_get_session_tickets(const mbedtls_ssl_config *conf);
```

Comment:
Does this need to be a public function?

Reply:
It is used outside ssl_tls.c. It could be a "static inline" function but then I would have to move the definition of  MBEDTLS_SSL_SESSION_TICKETS_TLS1_2/3_BIT/MASK in ssl.h. Can do if you prefer.

Reply:
I think we have `ssl_misc.h` for this kind of things (functions used in more than one C file but that don't need to be public).

Reply:
Ah yes I did not think about that, That's better, thanks. I will change it.